diesel_tests/build.rs

```diff
@@ -39,7 +39,7 @@ mod outer {
         let database_url = ::std::env::var("DATABASE_URL")
             .expect("DATABASE_URL must be set to run tests");
         let connection = PgConnection::establish(&database_url).unwrap();
-        let migrations_dir = migrations::find_migrations_directory().unwrap().join("postgresql");
+        let migrations_dir = migrations::find_migrations_directory().unwrap().join("../../migrations/postgresql");
```

Comment:
Without this change the dummy migrations from diesel_test are used(They are added in this commit)